### ISSUE TITLE
compilation error with poppler Annot

### ISSUE THREAD

**Issue by mnewt** [politza/pdf-tools#502]
It looks like maybe there is some incompatibility between the compiled version of poppler (0.79.0) on my macOS system and pdf-tools. It worked a couple months ago, which was the last time I tried to build it. Here is the relevant output. Can anyone offer tips to further troubleshoot?

```
$ g++ --version
Configured with: --prefix=/Applications/Xcode.app/Contents/Developer/usr --with-gxx-include-dir=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.14.sdk/usr/include/c++/4.2.1
Apple LLVM version 10.0.1 (clang-1001.0.46.4)
Target: x86_64-apple-darwin18.7.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
```

```
$ make clean all
...
...
...
g++ -Wall -Wall -I/usr/local/Cellar/glib/2.60.6/include/glib-2.0 -I/usr/local/Cellar/glib/2.60.6/lib/glib-2.0/include -I/usr/local/opt/gettext/include -I/usr/local/Cellar/pcre/8.43/include -I/usr/local/Cellar/poppler/0.79.0/include/poppler/glib -I/usr/local/Cellar/poppler/0.79.0/include/poppler -I/usr/local/Cellar/libffi/3.2.1/lib/libffi-3.2.1/include -I/usr/local/Cellar/glib/2.60.6/include -I/usr/local/Cellar/glib/2.60.6/include/glib-2.0 -I/usr/local/Cellar/glib/2.60.6/lib/glib-2.0/include -I/usr/local/opt/gettext/include -I/usr/local/Cellar/pcre/8.43/include -I/usr/local/Cellar/libpng/1.6.37/include/libpng16 -I/usr/local/Cellar/cairo/1.16.0_2/include/cairo -I/usr/local/Cellar/pixman/0.38.4/include/pixman-1 -I/usr/local/Cellar/fontconfig/2.13.1/include -I/usr/local/opt/freetype/include/freetype2 -I/usr/local/Cellar/poppler/0.79.0/include/poppler -I/usr/local/Cellar/libpng/1.6.37/include/libpng16 -std=c++11 -g -O2   -o epdfinfo epdfinfo-epdfinfo.o epdfinfo-poppler-hack.o -L/usr/local/Cellar/glib/2.60.6/lib -L/usr/local/opt/gettext/lib -lglib-2.0 -lintl -L/usr/local/Cellar/poppler/0.79.0/lib -L/usr/local/Cellar/glib/2.60.6/lib -L/usr/local/opt/gettext/lib -L/usr/local/Cellar/cairo/1.16.0_2/lib -lpoppler-glib -lgobject-2.0 -lglib-2.0 -lintl -lcairo -L -L/usr/local/Cellar/libpng/1.6.37/lib -lpng16 -lz libsynctex.a -lz  
ld: warning: directory not found for option '-L-L/usr/local/Cellar/libpng/1.6.37/lib'
Undefined symbols for architecture x86_64:
  "Annot::setAppearanceState(char const*)", referenced from:
      _xpoppler_annot_set_rectangle in epdfinfo-poppler-hack.o
  "Annot::setRect(double, double, double, double)", referenced from:
      _xpoppler_annot_set_rectangle in epdfinfo-poppler-hack.o
  "_pdfDocEncoding", referenced from:
      __xpoppler_goo_string_to_utf8 in epdfinfo-poppler-hack.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[1]: *** [epdfinfo] Error 1
make: *** [all] Error 2
```

**Comment by mnewt** [comment]
This is related to #480 and its fix, #490, shows the way.

I chased fixing individual libraries for a while before realizing that pkgconfig was missing some paths, and so couldn't find the libraries. I suspect that homebrew has changed some of the libraries and the pkgconfig installation itself. So in the event #490 isn't enough and the build process still isn't finding all the libraries, people can try something like:

    export PKG_CONFIG_PATH=/usr/lib/pkgconfig:/usr/local/share/pkgconfig:/usr/local/opt/pkgconfig:/usr/local/lib/pkgconfig:/usr/local/Homebrew/Library/Homebrew/os/mac/pkgconfig:/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Aliases/pkgconfig

**Comment by politza** [comment]
So, it would then be preferable if PKG_CONFIG_PATH was picked up from the
environment in autobuild.
